up.sh

```diff
@@ -40,14 +33,10 @@ echo -n > .bootstrap/bootstrapped_deployments
 if [[ "$*" == *tracing* ]]
 then
   set_docker_environment_value "TRACING_ENABLED" "true"
-else
-  set_docker_environment_value "TRACING_ENABLED" "false"
```

Comment:
This needs to be reinstated. If `tracing` is not deployed, we must force the env var to `false`, otherwise the gateway logs (and maybe others too) are full of errors.

Reply:
1. what kind of errors? wasnt aware of this issue
2. I removed it by mistake  but it should be set to false as follows:
`- TYK_GW_TRACER_ENABLED=${TRACING_ENABLED:-false}`

Reply:
so I checked and I haven't removed those 2 env vars. `-false` set them to false if they ar not defined or empty string. that's the whole point of this syntax.

Reply:
Run these commands to replicate error (run `./down.sh` if you need to):

```
./up.sh tracing
./down.sh
./up.sh
docker logs -f tyk-demo_tyk-gateway_1
```

You will see lots of errors, such as

```
2020/07/18 06:08:55 failed to send the request: Post http://zipkin:9411/api/v2/spans: dial tcp: lookup zipkin on 127.0.0.11:53: no such host
```

Reply:
Ok but regardless are you saying the value doesn’t get get? Does it happen with this PR or in general?